aws/DRVault-Single-Deployment.json

```diff
@@ -739,7 +739,12 @@
                         "waitAfterCompletion":"0",
                         "ignoreErrors":"false"
                      },
-                     "03-ChangeHostName":{
+                     "03-CheckVaultServiceStatus": {
```

Comment:
In DR PrivateArk Server should be stopped

Reply:
Fixed